starter-code/server.js

```diff
@@ -45,8 +58,13 @@ app.post('/articles', function(request, response) {
 
   function queryTwo() {
     client.query(
-      ``, // TODO: Write a SQL query to retrieve the author_id from the authors table for the new article
-      [], // TODO: Add the author name as data for the SQL query
+      // DONE: Write a SQL query to retrieve the author_id from the authors table for the new article
+      // estimate 5 min, actual2 min
+      `SELECT (author_id FROM authors WHERE author_id = $1, ;`,
```

Comment:
syntax again...this will give errors. check your parenthesis and the syntax at end of the line